WORKSPACE

```diff
@@ -283,11 +283,11 @@ http_archive(
         # TODO: Remove the fowllowing once librdkafka issue is resolved.
         """sed -i.bak '\|rd_kafka_log(rk,|,/ exceeded);/ s/^/\/\//' src/rdkafka_cgrp.c""",
```

Comment:
@yongtang any idea if that particular librdkafka issue is resolved? Not sure what the issue is.